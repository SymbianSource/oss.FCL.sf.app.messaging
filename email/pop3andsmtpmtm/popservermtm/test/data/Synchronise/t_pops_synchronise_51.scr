+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted
+OK 5 6540
+OK 5 messages
1 1314
2 1400
3 1275
4 1234
5 1317
.
-ERR UIDL not supported
-ERR Unknown Command "CAPA"
+OK TOP 5 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message007; Wed, 25 Jul 2001 11:44:38 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1106175; 25 Jul 2001 11:44 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6919643@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 12:59:49 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <PRD792A7>; Wed, 25 Jul 2001 12:48:37 +0100
Message-ID: <t_pops_synchronise_message007@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 7
Date: Wed, 25 Jul 2001 12:42:03 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Seven Seven Seven Seven Seven Seven Seven Seven Seven Seven  
.
+OK TOP 4 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message006; Wed, 25 Jul 2001 11:34:15 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2120593; 25 Jul 2001 11:34 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f68810d3@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 12:49:25 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
	id <PRD79H0X>; Wed, 25 Jul 2001 12:37:38 +0100
Message-ID: <t_pops_synchronise_message006@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 6
Date: Wed, 25 Jul 2001 12:31:03 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
	charset="iso-8859-1"

Six Six Six Six Six Six Six Six Six Six 
.
+OK TOP 3 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message005; Wed, 25 Jul 2001 11:33:55 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2119703; 25 Jul 2001 11:33 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f687c0fd@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 12:49:04 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
	id <PRD79H0W>; Wed, 25 Jul 2001 12:37:35 +0100
Message-ID: <t_pops_synchronise_message005@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 5
Date: Wed, 25 Jul 2001 12:31:00 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
	charset="iso-8859-1"

Five Five Five Five Five Five Five Five Five Five 
.
+OK TOP 2 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message004; Wed, 25 Jul 2001 11:33:44 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2119007; 25 Jul 2001 11:33 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f68770ab@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 12:48:44 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
	id <PRD79H04>; Wed, 25 Jul 2001 12:37:03 +0100
Message-ID: <t_pops_synchronise_message004@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 4
Date: Wed, 25 Jul 2001 12:30:28 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
	charset="iso-8859-1"

Four Four Four Four Four Four Four Four Four Four Four Four Four Four Four
.
+OK TOP 1 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message002; Wed, 25 Jul 2001 11:32:44 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2022711; 25 Jul 2001 11:32 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6866b8f@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 12:47:37 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
	id <PRD79H0P>; Wed, 25 Jul 2001 12:36:30 +0100
Message-ID: <t_pops_synchronise_message002@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 2
Date: Wed, 25 Jul 2001 12:29:55 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
	charset="iso-8859-1"

Two Two Two Two Two Two Two Two Two Two Two Two Two Two Two 
.
** Read cancelled
+OK delete 2 accepted 
+OK disconnecting 

