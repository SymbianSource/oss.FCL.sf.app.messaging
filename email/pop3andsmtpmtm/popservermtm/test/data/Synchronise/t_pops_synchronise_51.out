STAT
LIST
UIDL
CAPA
TOP 5 1
TOP 4 1
TOP 3 1
TOP 2 1
TOP 1 1
DELE 2
QUIT
